lib/utils/astutils.ml

```diff
@@ -44,6 +44,11 @@ let is_exp_const : exp -> bool = fun e ->
   | Pexp_constant _ -> true
   | _ -> false
 
+let is_exp_bool : exp -> bool = fun e ->
+  match e.pexp_desc with
```

Comment:
A small thing here - can you rewrite this using the `=|` operator, which handles matching on an expression an returning true if the provided expression on the left is a Pexp_construct node of a lident, e.g. `e =| "true" | e =| "false"`.